packages/spec-parser/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@wordpress/spec-parser",
```

Comment:
Honestly, I don't like this name. Spec parser? What spec? WordPress can build several parsers and not only for blocks. I'm thinking it should be `@wordpress/block-parser`.

Also, the `block-parser` can be misleading because it doesn't cover the whole block parsing, it's more `@wordpress/block-boundary-parser` or something like that.

I'm really bad at naming things so don't take my comment as a blocker or something, but more like a discussion point.

Reply:
> what spec?

the _Gutenberg_ spec. the PEG is the formal grammar and the `pegjs` parser is _the_ official implementation of that specification. the purpose of the PEG is to have a formal grammar that is both clear and which sets a standard against which other people can write their own implementations.

for example, CPython is the _spec_ Python implementation. it's slow, it's a beast, it doesn't have all the shinies that some other implementations have (static typing, better parallelization, faster optimizations) but we judge whether PyPy or Cython are _valid_ based on how completely they match the spec.

similarly here as people write their implementations we want to be able to compare those implementations and ask "is this complete?" and "is this valid?" in the traditional WordPress world these questions could only be answered by supplying a vast library of input and seeing if there were any differences in the output. _here_ with an actual specification we can define partial implementations and gauge completeness based on a real standard.

hope that wasn't rambling on and hope it clarified why I prefer the spec terminology. the PEGjs parser was always for me about providing a way to keep the grammar clear and at the forefront while providing a useful enough implementation to move the project along. I pity the fool who builds first a parser whose goal is performance and then expects people to walk backwards from the optimized code to try and determine what the actual rules and goals are.

Reply:
`@wordpress/block-serialization-spec` is fine by me even if verbose.

Reply:
Yeah I agree and would be confused by this package name. If we ship a `spec-parser` umbrella package in the future we can use the name then, but this should really be `block-spec` or `block-serialization-spec` as @youknowriad mentioned. This name is too vague and it would definitely confuse me.

Reply:
`@wordpress/block-serialization-spec-parser` - I will update tomorrow 👍